notebooks/emissions-units-with-pint.ipynb

```diff
@@ -111,7 +120,7 @@
    "source": [
     "## Pint Unit Registry\n",
     "\n",
-    "The `unit_registry` which sits underneath all conversions can be accessed via `UnitConverter`'s `unit_registry` property. Nevertheless, you should have no need to access it directly."
+    "The `unit_registry` which sits underneath all conversions can be accessed via `scmdata.units.UNIT_REGISTRY`or via `UnitConverter`'s `unit_registry` property."
```

Comment:
Needs updating because of getter pattern?

Reply:
Good pickup